test/libsolidity/syntaxTests/pragma/broken_version_2.sol

```diff
@@ -0,0 +1,3 @@
+pragma solidity pragma;
```

Comment:
@ekpyron The grammar file accepts this, while the parser (now) doesn't. Not sure what to do. Perhaps remove this test?

Reply:
Don't remove the test - if fixing the grammar is not an option, resp. more trouble than it's worth (and here that might be the case), then add the error id to the exclude list in `test/test_antrl*` (whatever the name of the script there)

Reply:
Thanks! I think it is more trouble, because the grammar correctly just assumes anything goes. This is a special case I'd say not worth the effort.

Reply:
Disabled. Let's see if anyone complains about the grammar not being super strict enough 😉

Reply:
The grammar isn't precise. What is the point even

Reply:
The point of the grammar or the point of the people wanting it to be 100% match?

Reply:
The point of the grammar (it was a joke)